modules/rbac-resource-definitions.adoc

```diff
@@ -6,15 +6,21 @@
 = Resource definitions
 
 [role="_abstract"]
-{product-title} includes multiple resources.
+{product-title} includes many resources.
 The following table lists the resources and describes the actions that users can perform with the `read` or `write` permission.
 
+[NOTE]
+====
+* To prevent privilege escalation, when you create a new token, your role's permissions limit you. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission.
```

Comment:
```suggestion
* To prevent privilege escalation, when you create a new token, your role's permissions limit the permission you can assign to that token. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission.
```